### PR TITLE
Unit test for CRUD of the harvester model

### PR DESCRIPTION
**Pull request by otoro90**

### CODE REVIEW

File: harvests/tests/test_models.py [hunk 1]
```diff
@@ -53,3 +53,11 @@ def test_create_harvester(self):
     def test_delete_harvester(self):
         self.harvester.delete()
         self.assertNotIn(self.harvester, self.queryset)
+
+    def test_update_harvester(self):
+        harvester = Harvester.objects.get(pk=10002)
+        self.assertEqual(harvester.name, 'Fabian')
```
Comment (danialeresbar) [inline harvests/tests/test_models.py:59]: These lines are not needed :)

File: harvests/tests/test_models.py [hunk 2]
```diff
@@ -53,3 +53,11 @@ def test_create_harvester(self):
...
+        Harvester.objects.filter(pk=10002).update(name='Oscar')
+        modified_harvester = Harvester.objects.get(pk=10002)
+        self.assertNotEqual(modified_harvester.name, 'Fabian')
+        self.assertEqual(modified_harvester.name, 'Oscar')
```
Comment (danialeresbar) [inline harvests/tests/test_models.py:63]: This line is not needed :)

**Review by danialeresbar — COMMENTED** [review]
Please check my suggestions :D 
Once you apply the changes we can proceed to integrate

**Comment by otoro90** [comment]
thanks for the comments, these have already been resolved
waiting your reply

**Review by danialeresbar — APPROVED** [review]
Nice!

**Pull request merged**